application/models/Files.php

```diff
@@ -216,4 +235,24 @@ public function getComments()
 							   ->exec();
 			return $results;
 		}
+		
+		public static function search($string)
+		{
+			if(empty($string))
+				return false;
+			
+			$searchArray = explode(' ', htmlspecialchars($string));
+			
+			$like = '%' . current($searchArray) . '';
+			$in = current($searchArray);
+			array_shift($searchArray);
```

Comment:
Le premier élément de $searchArray est stockée dans $like et le deuxième est dans $in ??
Et le array_shift, c'est pour pouvoir faire la boucle sans repasser par la première valeur (qui est déjà renseignée dans $like et $in).